### PR TITLE
Workflow Invoke

### PR DESCRIPTION
**Pull request by kraftp**

### CODE REVIEW

File: e-commerce/shop-backend/package.json [hunk 1]
```diff
@@ -3,16 +3,16 @@
   "version": "0.0.1",
   "scripts": {
     "build": "tsc",
-    "test": "npx knex migrate:rollback && npx knex migrate:up && npx knex seed:run && jest --detectOpenHandles",
-    "db:setup": "npx knex migrate:up && npx knex seed:run",
+    "test": "npx dbos rollback && npx dbos migrate && jest --detectOpenHandles",
+    "db:setup": "npx dbos migrate",
```
Comment (chuck-dbos) [inline e-commerce/shop-backend/package.json:7]: Does migrate always seed?

Comment (chuck-dbos) [reply e-commerce/shop-backend/package.json:7]: (That is to say, if you run with no preexisting database, are the products getting inserted into the store?

Comment (kraftp) [reply e-commerce/shop-backend/package.json:7]: Yes, this application is configured (in `dbos-config.yaml`) to always seed during migration.

Comment (chuck-dbos) [reply e-commerce/shop-backend/package.json:7]: This is not described in any documentation that comes up in a search for "seed".

**Review by chuck-dbos — APPROVED** [review]

**Pull request merged**